.github/pull_request_template.md

```diff
@@ -0,0 +1,96 @@
+
+
+### What Needs to Be Merged and Deployed Before this PR?
+
+This PR cannot be either merged or deployed until the following pre-requisite changes have been fully deployed:
+
+* CMSgov/some_repo#42
+
+<!--
+Add some items to the list above, or remove the entire section if it doesn't apply.
+
+Common items include:
+* Database migrations (which should always be deployed by themselves, to reduce risk).
+* New features in external dependencies (e.g. BFD).
+-->
+
+
+### Submitter Checklist
+
+Before requesting final review, I've gone through and verified that this PR complies with the following requirements:
+
+* [ ] I've refactored and rebased this PR (for help, see: [this](https://dev.to/maxwell_dev/the-git-rebase-introduction-i-wish-id-had) and [this](https://raphaelfabeni.com/git-editing-commits-part-1/), if needed, so that it is as small as it can reasonably be, in order to:
+    1. Ensure that any problems it causes have a small "blast radius".
+    2. Ensure that it'll be easier to rollback if that becomes necessary.
+    3. Ease the burden on reviewers.
+* [ ] I've verified that the commits in this PR:
+    1. Reasonably explain the "what" and "why" of the changes.
+    2. Leverage JIRA's [smart commits](https://confluence.atlassian.com/jirasoftwarecloud/processing-issues-with-smart-commits-788960027.html) to reference the JIRA ticket that they're associated with. For example, aim for commit messages like this (note also [the 50/72 formatting](https://stackoverflow.com/q/2290016) used here):
+        
+        ```
+        Added the whizbang to the doodad.
+
+        The new whizbang should really simplify things for our whoosit users.
+        It was a bit tricky to get it into the doodad, but we decided that the
+        flipwhee pattern was the best approach for now. Might want to revisit
+        that in the future if it ends up being too hard to maintain.
+
+        SOMEPROJECT-42
+        ```
+        
+* [ ] I've verified that this PR includes all required documentation changes, including `README` updates and changelog / release notes entries.
+* [ ] I've verified that all new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
+* [ ] I've verified that all tech debt and/or shortcomings introduced by this PR is detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
+* [ ] I've requested review from at least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
+* [ ] I've requested review from any relevant engineers on other projects (e.g. BFD, SLS, etc.).
+* [ ] I've verified that this PR and its changes comply with all other policies in the [DASG Engineering Standards](../policies/engineering_standards.md) and have specifically called out any/all deviations in this PR.
```

Comment:
@DeirdreHolub Or perhaps this?
```suggestion
* [ ] I've verified that this PR and its changes comply with all other policies in the [DASG Engineering Standards](../policies/engineering_standards.md) and have specifically called out any/all deviations in this PR.
    * Please review the standards every few months to ensure you're familiar with them.
```

Reply:
Why not directly ask PR writers to list deviations from the DASG engineering standard? It's a tone comment. Checklists should be easy to scan. The "I have verified this PR and its changes" statements make theses statement list items hard to scan and easy to automatically check.

Reply:
Excellent suggestion! I believe my recent changes address this concern.